src/Raven.Studio/wwwroot/Content/css/bootstrap/type.less

```diff
@@ -2,6 +2,10 @@
 // Typography
 // --------------------------------------------------
 
+.text-white {
```

Comment:
@kwiato do you think this is good location for this change? Does it look good in different themes? Usually we didn't have separate colors for icons but all shared same color.

Reply:
so I'm not sure we even need this class.

Reply:
in cloud we have a ` .text-emphasis` class

Reply:
The class has been removed